.github/workflows/actions.yml

```diff
@@ -6,28 +6,32 @@ on:
     - main
   pull_request:
 
+permissions:
+  contents: read
 
 jobs:
   gh-actions:
     runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
-        include:
-          - name: appauth
-          - name: dispatch
-          - name: github/checks
-          - name: github/mutex
-          - name: hashfiles
-          - name: jq
-          - name: retest
-          - name: torun
+        node-version:
+          - '20.x'
+        name:
+          - 'appauth'
+          - 'dispatch'
+          - 'github/checks'
+          - 'github/mutex'
+          - 'hashfiles'
+          - 'jq'
+          - 'retest'
+          - 'torun'
```

Comment:
least indent these please

im fussy about this becase config of any complexity quickly becomes unusable if these are not consistent and i think the lesser indent is generally preferable